src/app/onion/dashboard/dashboard.component.scss

```diff
@@ -38,4 +38,11 @@
     background: white;
     color: $light-blue;
   }
+}
+.loading-template {
+  max-width: $max-width;
+  padding: 20px;
+  box-sizing: border-box;
+  margin-top: -110px;
```

Comment:
Can we position this absolutely and remove the negative margin?

Reply:
I moved this up in order to load everything at once. I didn't want the tabs showing up and then flicking to have numbers next to them. 
![Screen Shot 2019-07-23 at 10 55 16 AM](https://user-images.githubusercontent.com/43140629/61722520-6ffaec00-ad38-11e9-8448-0d049e8f979e.png)